packages/cli/commands/project/download.js

```diff
@@ -36,6 +33,12 @@ exports.describe = i18n(`${i18nKey}.describe`);
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
+  const { projectConfig } = await getProjectConfig();
```

Comment:
If a project config already exists, should we abort the command and exit? It doesn't really make much sense to download a project within another project.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/907/commits/6907f42236eb9b7dc011ee7d35912316902208cf